### DISCUSSION TITLE
Being able to hold a keymap to repeatedly call it's action

### DISCUSSION THREAD

**Discussion by binamralamsal** [YaLTeR/niri#344] [Ideas]
I am sorry as I am not sure how to phrase this. I have a keymap like this:
```
XF86AudioRaiseVolume allow-when-locked=true { spawn "zsh" "-c" "~/.config/waybar/scripts/volume --inc"; }
XF86AudioLowerVolume allow-when-locked=true { spawn "zsh" "-c" "~/.config/waybar/scripts/volume --dec"; }
XF86AudioMute        allow-when-locked=true { spawn "zsh" "-c" "~/.config/waybar/scripts/volume --toggle"; }
XF86AudioMicMute     allow-when-locked=true { spawn "zsh" "-c" "~/.config/waybar/scripts/volume --toggle-mic"; }
XF86MonBrightnessUp  allow-when-locked=true { spawn "zsh" "-c" "~/.config/waybar/scripts/brightness --inc";
XF86MonBrightnessDown  allow-when-locked=true { spawn "zsh" "-c" "~/.config/waybar/scripts/brightness --dec"; }
```

It works fine but I want to hold a keymap to repeatedly call the action that is given there. Like, if I am holding a volume key then it should go on increasing the volume until I stop holding the volume key. Right now, if I am holding a keymap then it is fired once, I want to be able to repeatedly call it with specific interval while holding.

**Comment by YaLTeR** [comment]
Yeah, key repeat is something I have planned.

**Reply by salman-farooq-sh** [reply]
Is there any workaround meanwhile?
